config/llama_stack_local_config.yaml

```
@@ -0,0 +1,43 @@
+  model_id: sentence-transformers/all-mpnet-base-v2
+  provider_id: sentence-transformers
+  provider_model_id: sentence-transformers/all-mpnet-base-v2
+  model_type: embedding
+shields: []
+vector_dbs: []
+datasets: []
+scoring_fns: []
+benchmarks: []
+tool_groups: []
+server:
+  port: 8321 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)